arm-monitor/2015-04-01/swagger/autoscale_API.json

```diff
@@ -1012,15 +1024,19 @@
           "type": "array",
           "items": {
             "type": "integer",
-            "format": "int32"
+            "format": "int32",
```

Comment:
Whoops, nearly missed this! If hours can be a part of any request parameter, this could be a breaking change for your sdk. FYI